apps/cli/app/services.py

```diff
@@ -100,15 +104,28 @@ def update_contact(prev: Contact) -> None:
         console.print("🎉  Contact updated successfully!\n", end="\n." * 10)
 
 
+@with_confirmation
+def delete_contact(prev: Contact) -> None:
+    result = controller(
+        Actions.DELETE,
+        ContactPayload(name=prev.id),
+    )
+
+    if result == None:
+        console.print("🎉  Contact deleted successfully!\n", end="\n." * 10)
+
+
 @with_confirmation
 def get_birthdays_by_duration(*_) -> None:
     day_duration = prompt(
         "Enter days duration from today",
         error_message="Duration should be an Integer",
         optional=True,
     )
-
-    day_amount = day_duration if type(day_duration) == int else 7
+    try:
+        day_amount = int(day_duration)
+    except:
+        day_amount = 7
```

Comment:
the library from where we get this prompt util is already has int type validation, will add it later